<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>scroll2pdf</string>
  </property>
  <property name="windowIcon">
   <iconset theme="://img/scroll2pdf.png">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinOffsetY">
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Profile:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonDelete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QDoubleSpinBox" name="spinMaxError">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="labelErrorCurrent">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Offset Y:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2">
       <widget class="QPushButton" name="buttonSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinOffsetX">
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="spinSizeX">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Size X:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QLabel" name="labelPages">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="labelErrorLast">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Size Y:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QPushButton" name="buttonStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboProfile">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Offset X:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Max Error:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="16">
       <widget class="QGraphicsView" name="view"/>
      </item>
      <item row="11" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Error Current:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Pages:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="spinSizeY">
        <property name="minimum">
         <number>250</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QPushButton" name="buttonUndo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Error Last:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Frame time:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="2">
       <widget class="QLabel" name="labelFrameTime">
        <property name="text">
         <string>0 ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>comboProfile</tabstop>
  <tabstop>buttonDelete</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>spinOffsetX</tabstop>
  <tabstop>spinOffsetY</tabstop>
  <tabstop>spinSizeX</tabstop>
  <tabstop>spinSizeY</tabstop>
  <tabstop>spinMaxError</tabstop>
  <tabstop>buttonStart</tabstop>
  <tabstop>buttonUndo</tabstop>
  <tabstop>buttonSave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>spinOffsetX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>profileValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinOffsetY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>profileValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinSizeX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>profileValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinSizeY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>profileValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonUndo</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonUndo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinMaxError</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>profileValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboProfile</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>profileChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDelete</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>profileValueChanged()</slot>
  <slot>buttonStart()</slot>
  <slot>buttonUndo()</slot>
  <slot>buttonSave()</slot>
  <slot>profileChanged(int)</slot>
  <slot>buttonAdd()</slot>
  <slot>buttonDelete()</slot>
  <slot>profileNameChanged(QString)</slot>
 </slots>
</ui>
